news/9950.feature.rst

```diff
@@ -0,0 +1 @@
+Feature request github issue template has a section seeking willingness to contribute from the requester.
```

Comment:
The feature section is for features in pip itself; I think this should use `trivial`.

Reply:
Do you suggest I remove the news entry?

Reply:
No, just rename it to `9950.trivial.rst`.

Reply:
@uranusjr, I have made the suggested change in commit ce04e25